data/core/terrain-graphics.cfg

```diff
@@ -174,14 +174,18 @@ C*,K*,X*,Q*,W*,Ai,M*,*^V*,*^B*,_off^_usr#enddef
 {NEW:OVERLAY             *^Do                                               village/desert-oasis-3}
 
 # Swamp
-{OVERLAY_COMPLETE_LF     Ss        (!,Xv,Chs,!,C*,H*,M*,X*,Q*,A*,Ke,Kea,Kud*,I*) -85 base2      swamp/reed}
-{OVERLAY_COMPLETE_LF     Chs       (!,C*,K*,Ss)                              -85 base2      swamp/reed}
+{NEW:OVERLAY              Ss                                                swamp/reed LAYER=-85 FLAG=base2 ADJACENT="!,Xv,Chs,!,C*,H*,M*,X*,Q*,A*,Ke,Kea,Kud*,I*" }
+# {OVERLAY_COMPLETE_LF     Ss        (!,Xv,Chs,!,C*,H*,M*,X*,Q*,A*,Ke,Kea,Kud*,I*) -85 base2      swamp/reed}
```

Comment:
The old versions commented out should be removed entirely before this is merged. I realize this is still in draft though, so probably not a priority right now.

Reply:
I'm leaving a lot of that stuff in for now until deprecation is worked out.

Reply:
That's fine. As long as it's gone before it's merged.